srv/src/main/java/customer/capsidian/validators/NotesValidator.java

```diff
@@ -0,0 +1,48 @@
+package customer.capsidian.validators;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+import com.sap.cds.services.messages.Messages;
+
+import cds.gen.notesservice.Notes;
+
+@Component
+@PropertySource("classpath:ValidationMessages.properties")
+public class NotesValidator {
+    
+    @Value("${NO_DATA_MESSAGE}")
+    private String NO_DATA_MESSAGE;
+
+    @Value("${DATA_PATTERN_MESSAGE}")
```

Comment:
I didn't get it, what is the purpose to make this constant configured in *.properties? could you please explain?